modules/data.remote/R/GEDI_AGB_prep.R

```diff
@@ -129,23 +103,23 @@ GEDI_AGB_plot <- function(outdir, site.id, start_date, end_date) {
     PEcAn.logger::logger.info("The current point is outside of GEDI domain!")
     return(FALSE)
   } else {
-    if (!file.exists(file.path(site_folder, "extends.txt")) | !file.exists(file.path(site_folder, "GEDI_AGB.csv"))) {
+    if (!file.exists(file.path(site_folder, "extend.txt")) | !file.exists(file.path(site_folder, "GEDI_AGB.csv"))) {
```

Comment:
The requested change was from "extends" to "extent" (spatial domain) not "extend" (to make longer).

Reply:
Fixed.